### DISCUSSION TITLE
Snes games blurry

### DISCUSSION THREAD

**Discussion by abrusco** [eduardofilo/RG350_adam_image#82] [Ideas & Requests]
On the rg350 Nearest Neighbor scaling in snes games is  very blurry.  bicubic is fine. I think the retrocorp guide was wrong at that point.

**Comment by eduardofilo** [comment]
OK, I will check that. Thanks for the report.

**Comment by eduardofilo** [comment]
I have been testing and I am not able to find a good compromise that fits all the kind of screens supported by the image. I think that Bicubic makes the pixels very fuzzy. In addition to the issue of compromise, it is impossible to satisfy all tastes.
